starters/research-group/go.mod

```diff
@@ -5,5 +5,5 @@ go 1.15
 require (
 	github.com/wowchemy/wowchemy-hugo-themes/modules/wowchemy-plugin-netlify v1.0.0 // indirect
 	github.com/wowchemy/wowchemy-hugo-themes/modules/wowchemy-plugin-netlify-cms v1.0.0 // indirect
-	github.com/wowchemy/wowchemy-hugo-themes/modules/wowchemy/v5 v5.6.0 // indirect
+	github.com/wowchemy/wowchemy-hugo-themes/modules/wowchemy/v5 main // indirect
```

Comment:
This should be a specific version (commit or tag), not 'main' which will cause the site to break when contributors update Wowchemy.

Reply:
Which version should I put? 
Because the latest one is `v5.7.1-0.20230106134435-b9752d697a7b`, which does not include the `people` and `slider` v2 blocks, which are required for the updated version of the Research Group template.

Reply:
Yes, usually we would update the Wowchemy module in a dedicated PR first and then follow up with a PR to update the version of Wowchemy which templates use (as well as applying any breaking changes to the templates in the upgrade process).

In this case since you did a single PR to both add new features and update the templates, we can leave it as main, but please follow up after this PR is merged to fix the version to a specific version, otherwise everyone's site can break when contributors submit PRs with breaking changes to the main branch.